docs/tracing.rst

```diff
@@ -0,0 +1,144 @@
+.. _tracing:
+
+Tracing
+=======
+
+.. contents::
+    :depth: 3
+
+
+Overview
+--------
+
+It provides an instrument tracing wrapper function to use to instrument functions manually in pubtools-* projects.
+
+
+Usage
+.....................
+
+Set environment variables
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Following environment variables are used in the module:
+
+- ``OTEL_TRACING``: set ``true`` to enable instrument tracing, otherwise tracing is disabled.
+- ``OTEL_SERVICE_NAME``: required, set the value of the service.name resource attribute. It's
+  expected to be unique within the same namespace.
+
+
+OTEL Exporter
+~~~~~~~~~~~~~~
```

Comment:
Again, not perfectly aligned with the header text.

Reply:
Updated.